notebook_diff/test.ipynb

```diff
@@ -15,8 +15,8 @@
    "outputs": [],
```

Comment:
<p>Totally wrong, fix this here</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pettinato/poc_work/pull/4/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>